src/app/components/layout/footer/footer.component.html

```diff
@@ -10,7 +10,7 @@
         <a href="https://www.skycoin.net/skyminer">Skyminer</a>
         <a href="https://www.skycoin.net/team">Team</a>
         <a href="https://www.skycoin.net/blog">Blog</a>
-        <a href="/" class="active">Explorer</a>
+        <a href="/" class="active">Explorer API</a>
```

Comment:
The text should have been changed in the next link, not in this, since the next link is the one that open the api docs, while this one is marked as active and simply shows the browser.